src/App.tsx

```diff
@@ -1,24 +1,16 @@
 import React, { FC } from 'react';
-import logo from './logo.svg';
-import './App.css';
 
-const App: FC = () => (
-  <div className="App">
-    <header className="App-header">
-      <img src={logo} className="App-logo" alt="logo" />
-      <p>
-        Edit <code>src/App.tsx</code> and save to reload.
-      </p>
-      <a
-        className="App-link"
-        href="https://reactjs.org"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        Learn React
-      </a>
-    </header>
-  </div>
-);
+import { CssBaseline } from '@material-ui/core';
 
```

Comment:
I suggest removing this line.

Reply:
Resolved on 4847a19625a6936d6f19fb8a68131f0027c5db0e